vMenu/menus/VehicleOptions.cs

```diff
@@ -369,6 +384,9 @@ private void CreateMenu()
             }
             // always allowed
             menu.AddMenuItem(showHealth); // SHOW VEHICLE HEALTH
+            
+            // I don't really see why would you want to disable this so I will not add useless permissions
```

Comment:
:)

Reply:
Do you want me to delete this comment or add perms?

Reply:
no it's fine. you can leave this ;)